src/index.html

```diff
@@ -4,8 +4,26 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Wbpack Exercise</title>
+    <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <h1>Hello webpack!</h1>
+    <h1>Leaderboard</h1>
+    <main class="main">
+        <section class="scoresSection">
+            <div class="scoresTitle">
+                <h2>Recent Scores</h2>
+                <button id="refreshButton">Refresh</button>
+            </div>
+            <div class="scores" id="scoresList">
+            </div>
+        </section>
+        <section class="addScoreSection">
+            <h2>Add your Score</h2>
+            <input type="hidden" name="id" value="" id="id">
+            <input type="text" name="nameInput" id="nameInput" placeholder="Your name">
+            <input type="number" name="scoreInput" id="scoreInput" placeholder="Your Score">
```

Comment:
### Your Form
![form](https://user-images.githubusercontent.com/93039359/180629075-96991880-5a9f-4a0c-bbe2-e3dfad8515cc.png)

**[Optional]**

- [ ] It would be nice if your form will be cleared after the submission of the new entry. Because it is a normal behavior of a form and users are used to it :blush: Also, it shows your professionalism to your project :+1: